test/test_dcp.cpp

```diff
@@ -123,37 +123,37 @@ TEST_F (DcpTest, DcpRunTest)
    os_buf_t                *p_buf;
    int                     ret;
 
-   printf("\nGenerating mock set name request\n");
+   TEST_TRACE("\nGenerating mock set name request\n");
    p_buf = os_buf_alloc(PF_FRAME_BUFFER_SIZE);
    memcpy(p_buf->payload, set_name_req, sizeof(set_name_req));
    ret = pf_eth_recv(net, p_buf);
    EXPECT_EQ(ret, 1);
 
-   printf("\nGenerating mock set IP request\n");
+   TEST_TRACE("\nGenerating mock set IP request\n");
    p_buf = os_buf_alloc(PF_FRAME_BUFFER_SIZE);
    memcpy(p_buf->payload, set_ip_req, sizeof(set_ip_req));
    ret = pf_eth_recv(net, p_buf);
    EXPECT_EQ(ret, 1);
 
-   printf("\nGenerating mock set ident request\n");
+   TEST_TRACE("\nGenerating mock set ident request\n");
    p_buf = os_buf_alloc(PF_FRAME_BUFFER_SIZE);
    memcpy(p_buf->payload, ident_req, sizeof(ident_req));
    ret = pf_eth_recv(net, p_buf);
    EXPECT_EQ(ret, 1);
 
-   printf("\nGenerating mock factory reset request\n");
+   TEST_TRACE("\nGenerating mock factory reset request\n");
    p_buf = os_buf_alloc(PF_FRAME_BUFFER_SIZE);
    memcpy(p_buf->payload, factory_reset_req, sizeof(factory_reset_req));
    ret = pf_eth_recv(net, p_buf);
    EXPECT_EQ(ret, 1);
 
-   printf("\nGenerating mock flash LED request\n");
+   TEST_TRACE("\nGenerating mock flash LED request\n");
    p_buf = os_buf_alloc(PF_FRAME_BUFFER_SIZE);
    memcpy(p_buf->payload, signal_req, sizeof(signal_req));
    ret = pf_eth_recv(net, p_buf);
    EXPECT_EQ(ret, 1);
    /* 3.5 s should be enough, but on Windows even 5 s is not enough :-( */
```

Comment:
This comment does not longer apply, even if we build for Windows again (as we do not depend on operating system timers during test)? Also maybe try to reduce to 4 s.
Replace with: 
/* Wait for LED to flash three times at 1 Hz */

Reply:
Done